test/workbox-webpack-plugin/static/sw-src-define-plugin.js

```diff
@@ -0,0 +1,12 @@
+/*
+  Copyright 2018 Google LLC
```

Comment:
Nit: can you change this to 2020? It also helps to know at a glance when a file was added to the repo.

Reply:
done